### DISCUSSION TITLE
Announcement!!!

### DISCUSSION THREAD

**Discussion by emjay-engr** [emjay-engr/Open-Source-Group-3#1] [General]
@slickim @emjay-engr @D0cjay @Aeron-Jay-Tapia @Xiena12 @janin9 @Cix-prim

**Comment by emjay-engr** [comment]
hi huyss

**Reply by D0cjay** [reply]
Sa harong kita saaga ngani matapos na ang project HAHAHAH

**Reply by Xiena12** [reply]
matapos ba?

**Reply by emjay-engr** [reply]
tapos an kung matarabangan

**Reply by Cix-prim** [reply]
dae na nagrurun so code dahil kay aeron

**Reply by slickim** [reply]
otw
